airflow/providers/salesforce/hooks/salesforce.py

```diff
@@ -27,6 +27,11 @@
 import time
 from typing import Any, Dict, Iterable, List, Optional
 
+try:
+    from functools import cached_property
+except ImportError:
+    from cached_property import cached_property
+
```

Comment:
```suggestion
from airflow.compat.functools import cached_property
```

Reply:
One of the Pr's tests is failing `ModuleNotFoundError: No module named 'airflow.compat'`. Is it possible airflow.compat can't be used in one of the provider packages? The test that is failing is "Tests / Build and test provider packages wheel (pull_request)"

Reply:
yeah probably best to just switch that back.  i'll connect with jarek re guidance going forward.  thank you